kamailio/registrar-role.cfg

```diff
@@ -39,7 +39,6 @@ modparam("registrar", "received_avp", "$avp(AVP_RECV_PARAM)")
 modparam("registrar", "min_expires", REGISTRAR_MIN_EXPIRES)
 modparam("registrar", "max_expires", REGISTRAR_MAX_EXPIRES)
 modparam("registrar", "xavp_cfg", "regcfg")
-modparam("registrar", "reg_on_match_flag", 1)
 
```

Comment:
line 77
```
   $xavp(regcfg=>match_received) = $su;
    $xavp(regcfg[0]=>match_contact) = $(var(single_contact){nameaddr.uri});

    if ($sht(auth_cache=>$Au) != $null && registered("location", "$rz:$Au", 6) == 1) {
```
should be 
```
    $xavp(regcfg=>match_received) = $su;
    if($sht(auth_cache=>$Au) != $null && registered("location", "$rz:$Au", 2, 1) == 1 && $(xavp(ulattrs=>custom_channel_vars){s.len}) > 1) {
```
need to change registered function invocation to `registered("location", "$rz:$Au", 2, 1)` because of `reg_on_match_flag` change